yksmp/src/lib.rs

```diff
@@ -40,6 +40,10 @@ pub struct LiveVar {
 }
 
 impl LiveVar {
+    pub fn is_empty(&self) -> bool {
```

Comment:
What's this all about?

Reply:
Clippy complains if you have `len` without `is_empty`. Sometimes I disable that lint, but if it's easy to implement, it is a standard Rust convention so...

Reply:
Did you add the config file?

Reply:
ah, crate level.

Not what I was expecting, but OK.

If we end up needing that annotation in multiple crates later, let's use a config file.

Reply:
I think per-crate is slightly better myself, because it forces us to think a bit more about it. I don't feel strongly about it though.